src/segment-loader.js

```diff
@@ -1684,11 +1684,13 @@ export default class SegmentLoader extends videojs.EventTarget {
     }
 
     if (!this.handlePartialData_) {
-      if (this.startingMedia_.hasVideo && !segmentInfo.videoTimingInfo) {
+      const {hasAudio, hasVideo, isMuxed} = this.startingMedia_;
+
+      if (hasVideo && !segmentInfo.videoTimingInfo) {
         return false;
       }
 
-      if (this.startingMedia_.hasAudio && !segmentInfo.audioTimingInfo) {
+      if (hasAudio && !this.audioDisabled_ && !isMuxed && !segmentInfo.audioTimingInfo) {
```

Comment:
Might be worth adding a comment above saying that muxed content only relies on the video timing info right now.